### ISSUE TITLE
[Bug Report][3.3.11] V-tooltip doesn't work on mobile Android devices

### ISSUE THREAD

**Issue by demariaw** [vuetifyjs/vuetify#17949]
### Environment
**Vuetify Version:** 3.3.11
**Vue Version:** 3.3.4
**Browsers:** Chrome 115.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
The Tooltip component works correctly on iOS devices but not for Android devices. If you follow the Codepen on Chrome browser using an Android device such as Google Pixel 7 or Samsung Galaxy s22 and click the tooltip nothing happens. However these work for iOS devices.

### Expected Behavior
The tooltip should work for the button, icon and text on all devices.

### Actual Behavior
Tooltip worked for iOS devices such as iPhone 12 v14.1 but not for Android devices Google Pixel 7 or Samsung Galaxy s22 

### Reproduction Link
[https://codepen.io/Demaria-Woods/pen/OJadWBo](https://codepen.io/Demaria-Woods/pen/OJadWBo)

**Comment by lospoy** [comment]
Can confirm that the tooltip does not work on Android 13, using Chrome 115.0.5790.138

However, it _does_ work on Android 13, using Firefox 115.2.1

Perhaps a Chromium-specific issue and not Android? Did you use Chrome on iOS?

**Comment by demariaw** [comment]
> Can confirm that the tooltip does not work on Android 13, using Chrome 115.0.5790.138
> 
> However, it _does_ work on Android 13, using Firefox 115.2.1
> 
> Perhaps a Chromium-specific issue and not Android? Did you use Chrome on iOS?

@lospoy I am seeing it working as well on Android 13 with Firefox. I did use Chrome for iOS as well

**Comment by websitevirtuoso** [comment]
Yes the same on my android. it works on firefox but doesn't work on google chrome

**Comment by mostafaznv** [comment]
I can confirm that it doesnt work on macos too.

os: mac
browser: chrome (responsive tool)

**Comment by mostafaznv** [comment]
I can confirm that it doesnt work on macos too.

os: mac
browser: chrome (responsive tool)

**Comment by Pazza** [comment]
As of latest versions (vuetify and chrome) it still doesn't work. Doesn't work in vuetify's own docs.

**Comment by Jupdi** [comment]
Also not working for us, a fast solution would be great. Anyone having a workaround?

**Comment by xiscohv** [comment]
Do you happen to have any news on this bug? My web application is in production, and the tooltip fails on every Android device.
